logstash/pipeline/logstash.conf

```diff
@@ -1,7 +1,8 @@
 input {
-	tcp {
-		port => 5000
-	}
+    tcp {
+        port  => 5000
+        codec => json
```

Comment:
Agree, what about `extensions/logspout/README.md` instead of `extra/logspout/README.md` as you mentioned in the comment below ?